app/main.ipynb

```diff
@@ -151,17 +271,56 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
    "source": [
-    "# write your code here"
+    "df['datetime'] = pd.to_datetime(df['datetime'])\n",
+    "df['Order Day Of The Week'] = df['datetime'].dt.dayofweek\n",
+    "\n",
+    "days = {0: 'Monday', 1: 'Tuesday', 2: 'Wednesday', 3: 'Thursday', 4: 'Friday', 5: 'Saturday', 6: 'Sunday'}\n",
+    "df['Order Day Of The Week'] = df['Order Day Of The Week'].map(days)\n",
+    "\n",
+    "daily_income = df.groupby('Order Day Of The Week')['Item Price'].sum()\n",
+    "\n",
+    "plt.figure(figsize=(10, 10))\n",
+    "plt.bar(daily_income.index, daily_income.values)\n",
+    "plt.xlabel('Order Day Of The Week')\n",
+    "plt.ylabel('Income')\n",
+    "plt.title('Total Restaurant Income by Order Day Of The Week')\n",
+    "plt.show()"
```

Comment:
sort weekday columns by Monday-Sunday, so it's more human-readable and is more comfortable for a client
<img width="907" alt="image" src="https://github.com/mate-academy/py-restaurant-data-analysis/assets/61380547/2727e685-89a1-453b-bd38-2babdb58367e">